hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/StreamingGenerator.java

```diff
@@ -76,47 +77,60 @@ public Void call() throws Exception {
     generateBaseData();
```

Comment:
I guess we no longer need to generate base data in main thread.

Reply:
Yep, thanks. Removing it.